assets/js/blocks/reviews-by-product/block.js

```diff
@@ -1,283 +1,90 @@
-					<SelectControl
-						label={ __( 'Order Product Reviews by', 'woo-gutenberg-products-block' ) }
-						value={ attributes.orderby }
-						options={ [
-							{ label: 'Most recent', value: 'most-recent' },
-							{ label: 'Highest Rating', value: 'highest-rating' },
-							{ label: 'Lowest Rating', value: 'lowest-rating' },
-						] }
-						onChange={ ( orderby ) => setAttributes( { orderby } ) }
-					/>
-					<TextControl
-						label={ __( 'Reviews shown on page load', 'woo-gutenberg-products-block' ) }
-						value={ attributes.reviewsShown }
-						onChange={ ( reviewsShown ) => setAttributes( { reviewsShown } ) }
-						type="number"
-					/>
-				</PanelBody>
-			</InspectorControls>
-		);
-	}
-
-	renderProductControlItem( args ) {
-		const { item = 0 } = args;
-
-		return (
-			<SearchListItem
-				{ ...args }
-				countLabel={ sprintf(
-					_n(
-						'%d Review',
-						'%d Reviews',
-						item.rating_count,
-						'woo-gutenberg-products-block'
-					),
-					item.rating_count
-				) }
-				showCount
-				aria-label={ sprintf(
-					_n(
-						'%s, has %d review',
-						'%s, has %d reviews',
-						item.rating_count,
-						'woo-gutenberg-products-block'
-					),
-					item.name,
-					item.rating_count
-				) }
-			/>
-		);
-	}
-
-	renderEditMode() {
-		const { attributes, debouncedSpeak, setAttributes } = this.props;
-		const onDone = () => {
-			setAttributes( { editMode: false } );
-			debouncedSpeak(
-				__(
-					'Showing Reviews by Product block preview.',
-					'woo-gutenberg-products-block'
-				)
-			);
-		};
-
-		return (
-			<Placeholder
-				icon={ <ReviewsByProductIcon className="block-editor-block-icon" /> }
-				label={ __( 'Reviews by Product', 'woo-gutenberg-products-block' ) }
-				className="wc-block-reviews-by-product"
-			>
-				{ __(
-					'Show reviews of your product to build trust',
-					'woo-gutenberg-products-block'
-				) }
-				<div className="wc-block-reviews-by-product__selection">
-					<ProductControl
-						selected={ attributes.productId || 0 }
-						onChange={ ( value = [] ) => {
-							const id = value[ 0 ] ? value[ 0 ].id : 0;
-							setAttributes( { productId: id } );
-						} }
-						queryArgs={ {
-							orderby: 'comment_count',
-							order: 'desc',
-						} }
-						renderItem={ this.renderProductControlItem }
-					/>
-					<Button isDefault onClick={ onDone }>
-						{ __( 'Done', 'woo-gutenberg-products-block' ) }
-					</Button>
-				</div>
-			</Placeholder>
-		);
-	}
-
 	render() {
-		const { attributes, setAttributes } = this.props;
-		const { editMode } = attributes;
-		const { product } = this.state;
+		const { attributes } = this.props;
+		const { reviews } = this.state;
+		const { className, showProductRating, showReviewerName, showReviewerPicture, showReviewDate } = attributes;
+		const classes = classNames( 'wc-block-reviews-by-product', className, {
+			'has-picture': showReviewerPicture,
```

Comment:
Just a small note we should rename to avatar now this has approval.

Reply:
Added a note in #658 too.